t/lib/t/MusicBrainz/Server/Controller/User/Edit.pm

```diff
@@ -86,9 +86,9 @@ test 'Hide biography and website/homepage of beginners/limited users from not-lo
     $mech->get('/user/new_editor');
     html_ok($mech->content);
     $tx = test_xpath_html($mech->content);
-    $tx->ok('//tr[@class="biography"]/td[count(*)=1]/p[@class="deleted" and starts-with(text(), "This content is hidden to prevent spam.")]',
+    $tx->ok('//tr[@class="biography"]/td[count(*)=1]/div[@class="deleted" and starts-with(text(), "This content is hidden to prevent spam.")]',
         'biography field of beginner/limited user is hidden from not-logged-in user');
-    $tx->ok('//tr/td[preceding-sibling::th[1][normalize-space(text())="Homepage:"]]/p[@class="deleted" and starts-with(text(), "This content is hidden to prevent spam.")]',
+    $tx->ok('//tr/td[preceding-sibling::th[1][normalize-space(text())="Homepage:"]]/div[@class="deleted" and starts-with(text(), "This content is hidden to prevent spam.")]',
```

Comment:
I don’t notice any difference with vertical alignment here. Which browser was impacted?

By the way, commit message suggests biography used to be inside a `<p>` tag but it doesn’t seem to have ever been the case. The one change seems to actually be: The message about website/biography of beginner being hidden from visitor is displayed in a `<div>` tag instead of a `<p>` tag previously.

Reply:
I can reproduce this bug with Chrome and Firefox too, but for _Homepage_ field only.

_Biography_ field has a style workaround in [`root/static/styles/extra/user.less` on line 20](https://github.com/metabrainz/musicbrainz-server/blob/v-2019-10-17-hotfixes/root/static/styles/extra/user.less#L20-L22)

Reply:
Would you prefer to have the same style workaround for the homepage field then?

Reply:
No preference. I just wanted to mention it affects homepage only, not biography.